periphery/quoter.go

```diff
@@ -0,0 +1,165 @@
+package periphery
```

Comment:
just wonder from which do we get this file?

Reply:
from this: https://github.com/daoleno/uniswapv3-sdk/blob/master/periphery/quoter.go and our fork https://github.com/KyberNetwork/uniswapv3-sdk/blob/fix/Kep-185_upgrade_versionv040/periphery/quoter.go